src/vs/workbench/services/keybinding/browser/keybindingService.ts

```diff
@@ -535,6 +535,30 @@ export class WorkbenchKeybindingService extends AbstractKeybindingService {
 			return false;
 		}
 		const code = ScanCodeUtils.toEnum(event.code);
+
+		const NUMPAD_PRINTABLE_SCANCODES = [
```

Comment:
This should not be allocated every time the `mightProducePrintableCharacter` function runs.